scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala

```diff
@@ -916,9 +901,17 @@ sealed trait SCollection[T] extends PCollectionWrapper[T] {
    *   a new SCollection whose single value is an `Iterable` of the samples
    * @group transform
    */
-  def sample(sampleSize: Int): SCollection[Iterable[T]] = this.transform {
-    _.pApply(Sample.fixedSizeGlobally(sampleSize)).map(_.asScala)
-  }
+  // TODO move to implicit
```

Comment:
Do you want to resolve this in this PR?

Reply:
This would be binary breaking. it is a reminder for later. maybe worth an issue with milestone